File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a, b, c = map(int, input().split())

if a == b and b == c:
    print(10_000 + (a * 1000))
else:
    if a == b or b == c or a == c:
        print(1_000 + (a if a == b else c) * 100)
    else:
        print((max(a, c) if a > b else max(b, c)) * 100)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args)  {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        sc.close();

        if (a == b && b == c) 
            System.out.println(10_000 + (a * 1000));
        else
            if (a == b || b == c || a == c)
                System.out.println(1_000 + ((a == b ? a : c) * 100));
            else
                System.out.println((a > b ? Math.max(a, c) : Math.max(b, c)) * 100);
    }
}